apps/codeforafrica/src/lib/index.js

```diff
@@ -689,17 +691,12 @@ function getContactPageStaticProps() {
           ...getHero("contact"),
         },
         {
-          ...getContactForm(),
+          embedCode,
           slug: "contact-form",
         },
         {
           slug: "join-our-slack",
-          title: "We are on Slack!",
-          subtitle: "Join us",
-          action: {
-            label: "Join our Slack",
-            href: "https://docs.google.com/forms/d/e/1FAIpQLSdkfLU2yi2S1_7D27Z0I1TumkWy5brlam809Od9cc6CnXGA-A/viewform",
-          },
+          ...joinUs,
```

Comment:
Looking at this, join us and contact form are two separate sections/components. Why are we combining them in the CMS?

Reply:
Initially, they were bundled together in the Contact form, so I just simplified it. Let me separate them.